src/ui/ViewModels/OrganisationViewModelForAbout.cs

```diff
@@ -55,7 +55,7 @@ public void MergeIntoEnrichmentModel(ref InstitutionEnrichmentModel enrichmentMo
             var aboutTrainingProviders = new ObservableCollection<AccreditingProviderEnrichment>(
                 AboutTrainingProviders.Select(x => new AccreditingProviderEnrichment
                 {
-                    UcasInstitutionCode = x.InstitutionCode,
+                    UcasInstitutionCode = x.InstCode,
```

Comment:
Can i reject?

Reply:
no that's in the Enrichment which comes from API